tools/cli/pkg/command/reconciliation_info_test.go

```diff
@@ -93,6 +95,41 @@ func TestReconciliationOperationInfoCommand_Run(t *testing.T) {
 			},
 			wantErr: true,
 		},
+		{
+			name: "reconciliation info: happy path - reconciliation found",
+			fields: fields{
+				ctx:    testCtx,
+				output: outputJSON,
+				provideMshipClient: func(url string, _ *http.Client) (mothership.ClientInterface, error) {
+
+					r, err := ioutil.ReadFile("testdata/reconciliationInfoResponse.json")
+					if err != nil {
+						fmt.Print(err)
+					}
+					c, err := ioutil.ReadFile("testdata/configVersionResponse.json")
+					if err != nil {
+						fmt.Print(err)
+					}
+					m := msmock.NewMockClientInterface(ctrl)
```

Comment:
Do we check if mocks were called?

Reply:
nope. but the logic printing the result to the output is not accessible outside that command to unit test.
That would require some heavy refactor

Reply:
yes we do. The mocking lib assures that via `.Times(x)`

Reply:
I didn't notice the start of the test:
```
	ctrl := gomock.NewController(t)
	defer ctrl.Finish()
```